src/components/application_manager/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -278,11 +279,14 @@ void PerformInteractionRequest::onTimeOut() {
       break;
     }
     case mobile_apis::InteractionMode::VR_ONLY: {
-      application_manager_.updateRequestTimeout(
-          connection_key(), correlation_id(), default_timeout());
+      LOG4CXX_DEBUG(logger_, "Interaction Mode: VR_ONLY");
+      unsubscribe_from_event(hmi_apis::FunctionID::UI_PerformInteraction);
+      DisablePerformInteraction();
+      CommandRequestImpl::onTimeOut();
```

Comment:
@AGritsevich can you avoid duplicating CommandRequestImpl::onTimeOut(); and extract if from switch?